schemas/dataset-schema.json

```diff
@@ -217,23 +217,38 @@
-          "requirement_level": "required (often automatic)"
+          "title": "Title of the ETL dataset",
+          "description": "Title of the dataset (mostly for internal purposes, or for users of our data catalog) which is a one-line description of the dataset. NOTE: Dataset titles should be propagated automatically from snapshots. By default, the title of the dataset will be the title of the containing table. But, if there are multiple tables, or if the user wants to manually edit the dataset title, it can be done by editing `dataset.title`.",
+          "requirement_level": "required (often automatic)",
+          "guidelines": [
+            ["Must start with a capital letter."],
+            ["Must not end with a period."],
+            ["Should identify the dataset (i.e. the collection of tables resulting from one or more original data products)."],
+            ["If the dataset contains only one table, use its title (which usually coincides with the title of a data product or a snapshot)."]
+          ]
         },
         "description": {
           "type": "string",
-          "title": "Description of this ETL dataset",
-          "description": "Description of the dataset (mostly for internal purposes, or for users of our data catalog) which is a one- (or a few) paragraph description of the dataset (follow the guidelines of the origin's `description`). NOTE: Dataset descriptions should be propagated automatically from snapshots. By default, the description of the dataset will be the description of the containing table. But, if there are multiple tables, or if the user wants to manually edit the dataset description, it can be done by editing `dataset.description`.",
-          "requirement_level": "recommended (often automatic)"
+          "title": "Description of the ETL dataset",
+          "description": "Description of the dataset (mostly for internal purposes, or for users of our data catalog) which is a one- (or a few) paragraph description of the content of the tables. NOTE: Dataset descriptions should be propagated automatically from snapshots. By default, the description of the dataset will be the description of the containing table. But, if there are multiple tables, or if the user wants to manually edit the dataset description, it can be done by editing `dataset.description`.",
+          "requirement_level": "recommended (often automatic)",
+          "guidelines": [
+            ["Must start with a capital letter."],
+            ["Must end with a period."],
+            ["Must not mention other metadata fields like `producer`.", {"type": "exceptions", "value": ["Other metadata fields are crucial in the description of the data product."]}],
```

Comment:
I would phrase it like

"Must not mention other metadata fields (e.g. `producer` or ...)." (add another metadata field)

and then

"The other metadata fields are crucial in describing the data product."